src/pwd/pwd.rs

```diff
@@ -15,15 +15,28 @@ extern crate getopts;
 extern crate uucore;
 
 use std::env;
+use std::path::{Path, PathBuf};
+use std::io;
 
 static NAME: &'static str = "pwd";
 static VERSION: &'static str = env!("CARGO_PKG_VERSION");
 
+pub fn absolute_path(path: &Path) -> io::Result<PathBuf> {
+    let path_buf = path.canonicalize()?;
+
+    #[cfg(windows)]
+    let path_buf = Path::new(path_buf.as_path().to_string_lossy().trim_left_matches(r"\\?\")).to_path_buf();
```

Comment:
Does this actually work for all UNC paths?  I don't really know how they work.  What I meant before was modifying the test itself to check for UNC paths rather than normal ones on Windows.

Reply:
@Arcterus didn't tested it honestly, stole from some large project

Reply:
Welp, okay then lol.  I'll just assume it's fine for now.